### ISSUE TITLE
[portus] FQDN env var not being passed to Apache

### ISSUE THREAD

**Issue by ntwrkguru** [openSUSE/docker-containers#45]
Despite setting the `PORTUS_MACHINE_FQDN_VALUE` env var, Apache is not picking it up.

```bash
AH00558: httpd-prefork: Could not reliably determine the server's fully qualified domain name, using 192.168.66.3. Set the 'ServerName' directive globally to suppress this message
[Tue Mar 14 23:58:42.358284 2017] [ssl:warn] [pid 32] AH01909: 192.168.66.3:443:0 server certificate does NOT include an ID which matches the server name
```

**Comment by flavio** [comment]
ping @mssola

**Comment by tturbs** [comment]
Also it is failed to set SSL directories.

**Comment by mssola** [comment]
@ntwrkguru could you try with #69.

> @tturbs  Also it is failed to set SSL directories.

What do you mean ? Could you explain your exact problem ?

In general, I'd switch to a non-Apache image (like `head`), since we have quite some examples with it (see [this](https://github.com/SUSE/Portus/tree/master/examples)).

**Comment by mssola** [comment]
Fixed by #69
